docs/getting-started/foundation/Foundation.stories.mdx

```diff
@@ -45,22 +45,22 @@ A foundation CSS variable can be accessed in a CSS class:
 
 ```css
 .button {
-    color: var(--o-ui-primary-50);
+    color: var(--o-ui-alias-primary-50);
```

Comment:
Does "primary" still belong in foundation? It seem's theme related to me. It's not a token.